### DISCUSSION TITLE
bookmark/favorites

### DISCUSSION THREAD

**Discussion by Tealk** [GitFiend/Support#113] [Ideas]
Hello

would it be possible that repositorys can be pinned? So you could work nicer with a larger quantity and still quickly get to the often needed repositorys.

**Comment by GitFiend** [comment]
I've been working on a multi repo feature that would allow some pinning. This would let you open a folder containing repos, and you would be able to pin the repos you wanted to track to the top of the list, see status as well as apply operations like pull to all.

Would something like this help?

Here's a demo of switching between repos (no pinning yet):

https://user-images.githubusercontent.com/47970508/148464062-a1b00a71-5dae-4f76-9eaf-39e8235d6272.mp4

**Reply by Tealk** [reply]
I have already activated the function, I think it's great!

But that doesn't quite help me, I organize projects into folders and projects have multiple repositories. I open these folders with the Workspaces function. But I would like to pin these workspaces

e.g. I often need the workspace ButterflyAspect and would like to pin it on top so that it doesn't fall out of the history.
![grafik](https://user-images.githubusercontent.com/12276250/148546512-7fc47313-5555-4685-9810-0a4766bf7708.png)

**Reply by GitFiend** [reply]
Thanks for more details. I do think some way of marking your favourite repos would be helpful.
